lib/aruba/api.rb

```diff
@@ -729,6 +735,61 @@ def io_wait
       @aruba_io_wait_seconds || DEFAULT_IO_WAIT_SECONDS
     end
 
+    DEFAULT_ROOT_DIRECTORY = Dir.getwd
+
+    # The root directory of aruba
+    def root_directory
```

Comment:
I see, but it makes the configuration of the fixtures more cleaner. I added a new PR for this. Would you mind to merge it first, then?